server/plugin/api.go

```diff
@@ -984,10 +987,24 @@ func (p *Plugin) getYourAssignments(c *UserContext, w http.ResponseWriter, r *ht
 	result, _, err := githubClient.Search.Issues(c.Ctx, query, &github.SearchOptions{})
 	if err != nil {
 		c.Log.WithError(err).With(logger.LogContext{"query": query}).Warnf("Failed to search for assignments")
-		return
+		return nil
```

Comment:
Let's return an empty array instead here, or have the parent function set an empty array if this is `nil`. We don't want to provide nil data to the client like this